doc/project/checklist.md

```diff
@@ -111,6 +111,19 @@ FPGA synthesis timing meet (Fmax-10%) target or better
 CDC Sync flops use behavioral synchronization macros(`prim_flop_2sync`) not
 2flops.
 
+### SEC_CM_IMPLEMENTED
```

Comment:
Are we planning to use SEC_CM_IMPLEMENTED to check against general security implementation guidance?

For example, will review of IP level resets fall in this category?